src/stickyfill.js

```diff
@@ -317,6 +321,16 @@ const Stickyfill = {
     stickies,
     Sticky,
 
+    forceSticky () {
+        seppuku = false;
+
+        if (!isInitialized) {
```

Comment:
Let’s do the check inside the `init` function. This will allow the `init` function to be called from somewhere else without the need for additional checks. A little more future proof.